packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -57,19 +63,19 @@ export class MetadataOutlineProvider
       return Promise.resolve([org]);
     }
 
-    if (!element.children) {
-      switch (element.type) {
-        case NodeType.Org:
-          element.setChildren(await this.getTypes(), NodeType.MetadataType);
-          break;
-        case NodeType.Folder:
-        case NodeType.MetadataType:
-          const type = TypeUtils.FOLDER_TYPES.has(element.fullName)
-            ? NodeType.Folder
-            : NodeType.MetadataCmp;
-          element.setChildren(await this.getComponents(element), type);
-          break;
-      }
+    switch (element.type) {
+      case NodeType.Org:
+        element.setChildren(await this.getTypes(), NodeType.MetadataType);
+        this.toRefresh = false;
+        break;
+      case NodeType.Folder:
+      case NodeType.MetadataType:
+        const type = TypeUtils.FOLDER_TYPES.has(element.fullName)
+          ? NodeType.Folder
+          : NodeType.MetadataCmp;
+        element.setChildren(await this.getComponents(element), type);
+        element.toRefresh = false;
+        break;
```

Comment:
Do we need a `default` here ?

Reply:
No other node type can be expanded and have it's children fetched, so not necessarily.